Add image to home

peterdesmet opened the pull request.

peterdesmet commented:
@pkaczensky Where do you want the image to appear in relation to the text on the homepage?

pkaczensky commented:
@peterdesmet if possible I would add the copyright to the image, not the caption. On my mobile phone view, there is not even a caption visible. 

I saw your email concerning the copyright specifics. We will have to ask the image authors again if they agree as we did not explicitly ask for this froviding the link to the gallery. I will send an email concerning this topic.

pkaczensky commented:
@peterdesmet Sorry, this was the wrong comment - there are too many open requests...

Concerning the graphic on the page "Home", I would position it either next to the text "What do we want to do".
Under the text it becomes too big.

peterdesmet commented:
> @peterdesmet if possible I would add the copyright to the image, not the caption. On my mobile phone view, there is not even a caption visible.

This is now fixed and captions will be visible on small screens: https://wildlifecamera.eu/gallery/

> I would position it either next to the text "What do we want to do".

Done, looks like this. I have removed the side menu, since it didn't add much and looks better with all content centralized. I have assigned you as reviewer.

![BIG_PICTURE-04-10-2025_02_18_PM](https://github.com/user-attachments/assets/e542aa15-1605-4e9a-912a-e0062dd8d01f)

peterdesmet commented:
Conflicts can happen if the file (in this case `home.md`) was changed to much in `main` while this PR was open. No worries, I can resolve it.
